src/prob/pf.jl

```diff
@@ -334,7 +332,7 @@ end
 
 Calculates the voltage from PowerFlowData struct.
 """
-function _get_v(pfd::PowerFlowData, Vp::Vector{Complex{Float64}}, n::Tuple{Int64, Int64})
+function _get_v(pfd::PowerFlowData, Vp::Vector{Complex{Float64}}, n::Union{Tuple{Int64, Int64},Int64} )
```

Comment:
I think Carleton/David would prefer multiple dispatch here, not union types, because multiple dispatch is faster than union types.